src/domain/settlementWindow/index.js

```diff
@@ -46,12 +46,10 @@ module.exports = {
       if (settlementWindows && settlementWindows.length > 0) {
         return settlementWindows
       } else {
-        const err = new Error(`settlementWindow by filters: ${JSON.stringify(params.query).replace(/"/g, '')} not found`)
-        throw err
+        throw ErrorHandler.Factory.createInternalServerFSPIOPError(`settlementWindow by filters: ${JSON.stringify(params.query).replace(/"/g, '')} not found`)
       }
     } else {
-      const err = new Error('Use at least one parameter: participantId, state, fromDateTime, toDateTime')
-      throw err
+      throw ErrorHandler.Factory.createInternalServerFSPIOPError('Use at least one parameter: participantId, state, fromDateTime, toDateTime')
```

Comment:
Perhaps this could be a 3100 generic validation error or a 3102 Missing mandatory element error?

Reply:
Set to Validation error (3100)